app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html

```diff
@@ -178,169 +178,173 @@ <h2 class="moj-task-list__section">
-                </span>
-              {% endif %}
-            </li>
-          </ul>
-        </li>
+        {% if data["found-interventions"] %}
+          <li>
+            {% if data["accommodation-more-info-completed"] %}
+              <strong class="govuk-tag govuk-tag app-task-list__tag">
+                completed
+              </strong>
+            {% elseif data["accommodation-sentence"] %}
+              <strong class="govuk-tag govuk-tag app-task-list__tag">
+                in progress
+              </strong>
+            {% elseif data["needs-and-requirements-completed"] %}
+              <strong class="govuk-tag govuk-tag--grey app-task-list__tag">
+                not started
+              </strong>
+            {% else %}
+              <strong class="govuk-tag govuk-tag--grey app-task-list__tag">
+                cannot start yet
+              </strong>
+            {% endif %}
+            <h2 class="moj-task-list__section">
+              <span class="moj-task-list__section-number">5.
+              </span>
+              Accommodation referral
+            </h2>
+            <ul class="moj-task-list__items">
+              <li class="moj-task-list__item">
+                {% if data["needs-and-requirements-completed"] %}
+                  <a class="moj-task-list__task-name" href="accommodation-service/select-sentence">Select relevant sentence</a>
+                {% else %}
+                  <span class="app-task-list__task-name">
+                    Select relevant sentence
+                  </span>
+                {% endif %}
+              </li>
+              <li class="moj-task-list__item">
+                {% if data["accommodation-sentence"] %}
+                  <a class="moj-task-list__task-name" href="accommodation-service/desired-outcomes">Select desired outcomes</a>
+                {% else %}
+                  <span class="app-task-list__task-name">
+                    Select desired outcomes
+                  </span>
+                {% endif %}
+              </li>
+              <li class="moj-task-list__item">
+                {% if data["accommodation-desired-outcomes"] %}
+                  <a class="moj-task-list__task-name" href="accommodation-service/complexity-level">Select required complexity level</a>
+                {% else %}
+                  <span class="app-task-list__task-name">
+                    Select required complexity level
+                  </span>
+                {% endif %}
+              </li>
+              <li class="moj-task-list__item">
+                {% if data["accommodation-desired-outcomes"] %}
+                  <a class="moj-task-list__task-name" href="accommodation-service/completion-date">What date does the accommodation service need to be completed by?</a>
+                {% else %}
+                  <span class="app-task-list__task-name">
+                    What date does the accommodation service need to be completed by?
```

Comment:
This is also inconsistent with the social inclusion one (which just says "Enter completion date")

Reply:
👍 changed